hack/istio/multicluster/install-multi-primary.sh

```diff
@@ -136,7 +136,37 @@ if [ "${MANAGE_KIND}" == "true" ]; then
     docker network create kind || true
 
     # Given: 172.18.0.0/16 this should return 172.18
-    beginning_subnet_octets=$(docker network inspect kind --format '{{(index .IPAM.Config 0).Subnet}}' | cut -d'.' -f1,2)
+    subnet=""
```

Comment:
Why is this change to a multi-cluster hack script in this PR?

Reply:
It's just something I fixed while working on and testing this.  Nick and I already talked about it and decided to leave it in as opposed to having to create another PR.  If you are opposed I suppose I could remove it and make it it's own thing. It's just a bonus :-)

Reply:
that's fine..just making sure you didn't have a change that you didn't mean to put in the  PR